parsers/CL.py

```diff
@@ -59,6 +70,38 @@ def data_processor(raw_data):
     return ordered_data
 
 
+def get_data_live(session, logger):
+    """Requests generation data in json format."""
+
+    s = session or requests.session()
+    json_data = s.get(API_BASE_URL_LIVE).json()
+    return json_data
+
+
+def production_processor_live(json_data, zone_key):
+    """
+    Extracts data timestamp and sums regional data into totals by key.
+    Maps keys to type and returns a tuple.
+    """
+
+# unsure about this time object, but seems to be unix time, but in msecs
+        # 1) is this right?
+        # 2) does this need to have a timezone or anything??
+    dt = arrow.get(json_data['fecha']/1000)
```

Comment:
```suggestion
    dt = arrow.get(json_data['fecha']/1000, tzinfo='Chile/Continental')
```
This will add the correct timezone to the datetime object.